Makefile

```diff
@@ -29,17 +30,17 @@ HANDLER_PULL_POLICY ?= Always
 OPERATOR_PULL_POLICY ?= Always
 export IMAGE_BUILDER ?= docker
 
-WHAT ?= ./pkg ./controllers
+WHAT ?= ./pkg/... ./controllers/...
 
-unit_test_args ?=  -r -keepGoing --randomizeAllSpecs --randomizeSuites --race --trace $(UNIT_TEST_ARGS)
+unit_test_args ?=  -r -keep-going --randomize-all --randomize-suites --race --trace $(UNIT_TEST_ARGS)
```

Comment:
about `--randomize-all --randomize-suite`
aint better to have a deterministic run order? (as we are doing on kubevirt by fixing the seed)
(can be on a follow PR if desired)

Reply:
ye i understand, just on kubevirt (at least e2e) we are doing it ordered,
anyhow we have less flakes and it is unit tests so it is less problematic + whatever you prefer

Reply:
e2e is another kind of anymal.